Makefile.am

```diff
@@ -1,5 +1,3 @@
-AUTOMAKE_OPTIONS = foreign

```

Comment:
as stated in the last PR, modern Automake reads this from `configure.ac`, not Makefile.am